### ISSUE TITLE
Causes Fatal Error on windows phone 8 (IE11)

### ISSUE THREAD

**Issue by aintgoin2goa** [Financial-Times/o-tracking#81]
For some reason that I don't fully understand the way in which data is persisted in local storage seems to fail in IE11 on windows phone (and possible desktop - haven't checked)

When the queue is created it is saved as a HTML Encoded representation of an empty array ("%5B%5D" rather than "[]").  Therefore, this code (store.js line 138)
```
if (loadStore) {
		try {
			this.data = JSON.parse(loadStore);
		} catch (error) {
			utils.broadcast('oErrors', 'log', {
				error: error,
				module: 'o-tracking'
			});
			this.data = loadStore;
		}
	}
```
causes the queue to be set to "[]" rather than [].

This means that in this bit (queue.js, line 100):
```
function doAdd(item) {
		self.queue.push({
			created_at: (new Date()).valueOf(),
			id: utils.guid(),
			item: item
		});
	}
```
self.queue is a string not an array.  Therefore the whole operation fails.

**Comment by adambraimbridge** [comment]
http://i.imgur.com/lBzg3NF.gif

**Comment by markstephens** [comment]
This should be fixed with version 1.0.17
